_posts/2020-11-16-october-rubygems-update.md

```diff
@@ -0,0 +1,56 @@
+---
+title: October 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems, Ruby Toolbox and RubyGems.org in September.
+
+## RubyGems News
+
+In October we released a new Bundler version --- **`2.2.0.rc.2`**. We also upgraded Bundler's & Rubygems' vendored Molinillo to **`0.7.0`**  (it's latest release). - [#3402](https://github.com/rubygems/rubygems/pull/3402), [#3388](https://github.com/rubygems/rubygems/pull/3388). 
```

Comment:
I'd highlight "In October, we released Bundler 2.2.0.rc.2 and rubygems 3.2.0.rc.2", and move the Molinillo upgrade to the list of other improvements.

Reply:
Thank you @deivid-rodriguez.